### PR TITLE
html: form validator

### PR DESCRIPTION
**Pull request by slavidichkov**

### CODE REVIEW

File: src/com/clouway/task5/formValidator.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    var password = document.forms["myForm"]["password"].value;
+    var isValid=true;
+    var nameValidator=new RegExp("^([a-z\A-Z]){3,15}$");
+    var egnValidator=new RegExp("^([0-9]){10,10}$");
+    var ageValidator=new RegExp("^(1[89]|[2-9][0-9]|1[10][0-8])$");
+    var addressValidator=new RegExp("^([a-z\A-Z\0-9]){1,100}$");
+    var passwordValidator=new RegExp("^([a-z\A-Z\0-9]){6,18}$");
+
+    if (!nameValidator.test(fname)) {
+        setVisibility("wrongFname", "visible");
```
Comment (tsony-tsonev) [inline src/com/clouway/task5/formValidator.js]: you can create two functions hidden('elementId') and vsible('elementId') this way you won't need to now what value you should  pass for the css property 'visibility'

File: src/com/clouway/task5/validateForm.css [hunk 2]
```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: #eee;
+}
+.box{
+
+}
```
Comment (tsony-tsonev) [inline src/com/clouway/task5/validateForm.css]: no styles you can delete this

File: src/com/clouway/task5/validateForm.css [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    color: red;
+}
+
+.visible{
+    visibility: visible;
+}
+
+.invisible{
+    visibility: hidden;
+}
```
Comment (tsony-tsonev) [inline src/com/clouway/task5/validateForm.css]: this two I think are no longer used you can delete them

File: src/com/clouway/task5/validateFormFromTask3.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+                        <input type="password" id="conformPassword" placeholder="Conform Password" class="form-control">
+                    </div>
+                </div>
+                <br>
+                <div >
+                    <div>
+                        <input type="submit" value="Submit" class="btn btn-primary btn-block">
+                    </div>
+                </div>
+            </form> <!-- /form -->
```
Comment (tsony-tsonev) [inline src/com/clouway/task5/validateFormFromTask3.html]: commented html

**Comment by tsony-tsonev** [comment]
Nice form btw you have experimented with the bootstrap

File: src/com/clouway/task5/validateForm.css (hunk 3 above)
Comment (slavidichkov) [inline src/com/clouway/task5/validateForm.css]: done

File: src/com/clouway/task5/validateForm.css (hunk 2 above)
Comment (slavidichkov) [inline src/com/clouway/task5/validateForm.css]: done

File: src/com/clouway/task5/validateFormFromTask3.html (hunk 4 above)
Comment (slavidichkov) [inline src/com/clouway/task5/validateFormFromTask3.html]: fixed

File: src/com/clouway/task5/formValidator.js (hunk 1 above)
Comment (slavidichkov) [inline src/com/clouway/task5/formValidator.js]: done

File: src/com/clouway/task5/formValidator.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+    if (!passwordValidator.test(password)) {
+        visible("wrongPassword");
+        isValid=false;
+    }else{
+        hidden("wrongPassword");
+    }
+    return isValid;
+}
+
+function  hidden(elementId){
```
Comment (tsony-tsonev) [inline src/com/clouway/task5/formValidator.js]: hideElement

File: src/com/clouway/task5/formValidator.js [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+    var element = document.getElementById(elementId);
+    element.style.visibility = 'hidden';
+}
+
+function visible(elementId){
```
Comment (tsony-tsonev) [inline src/com/clouway/task5/formValidator.js]: showElement

**Comment by tsony-tsonev** [comment]
LGTM you can merge after you rename the commented methods

File: src/com/clouway/task5/formValidator.js (hunk 5 above)
Comment (slavidichkov) [inline src/com/clouway/task5/formValidator.js]: fixed

**Pull request merged**